src/components/ColorMode/ColorMode.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { render } from '@testing-library/react';
+import * as React from 'react';
+import { Button } from '../Button/Button';
+import { ColorMode } from './ColorMode';
+
+const renderColorMode = ({ dark }: { dark: boolean }) =>
+    render(
+        <ColorMode dark={dark}>
+            <Button>Click</Button>
+        </ColorMode>
+    );
+
+describe('ColorMode', () => {
+    it('renders a div wrapper with light-mode by default', () => {
```

Comment:
Why does it say "by default" when we are explicitly passing `dark: false`?

```suggestion
    it('renders a div wrapper with light-mode if we pass dark prop as false', () => {
```